skyvern-frontend/.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+cd skyvern-frontend
+
+npm run precommit
```

Comment:
will this cause issues if we already have pre-commit running?

Reply:
This will run the script named `precommit` inside `package.json` in `skyvern-frontend` directory, so I don't think there is a conflict.

Reply:
hm, can (should?) we include this in our current pre-commit config?

Sth like this within `.pre-commit-config.yaml`
```
.
.
.
  - repo: local
    hooks:
      - id: alembic-check
        name: Alembic Check
        entry: ./run_alembic_check.sh
        language: script
        stages: [ commit ]
  - repo: local
    hooks:
      - id: frontend-precommit
        name: Frontend Pre-commit Hook
        entry: sh -c 'cd skyvern-frontend && npm run precommit'
        language: system
        files: '\.(js|jsx|ts|tsx)$'
        stages: [commit]
```

Reply:
The script will have to include `npm install` as well to install `eslint` and `prettier`. Husky precommit hook is lazily set up inside the frontend directory once `npm install` is run.